UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -61,6 +61,8 @@ public class UniversalObjectPhysics : NetworkBehaviour, IRightClickable
 
 	[PrefabModeOnly] public bool Intangible = false;
 
+	[PlayModeOnly] public bool CanNotBeWindPushed = false;
```

Comment:
Booleans with `Not` in their names tend to cause confusion. Could it be something like `WindImmunity` instead?

If it were inverted, it could just be called `CanBeWindPushed`.

Reply:
find inverted would also be more confusing maybe?

Reply:
I think `CanBeWIndPushed` is more clear than `CanNotBeWindPushed == false`, in the second case you have two negations.

Reply:
agree